pandas/plotting/_core.py

```diff
@@ -2000,7 +2000,7 @@ def maybe_color_bp(bp):
 
     def plot_group(keys, values, ax):
         keys = [pprint_thing(x) for x in keys]
-        values = [remove_na_arraylike(v) for v in values]
+        values = [np.asarray(remove_na_arraylike(v)) for v in values]
         bp = ax.boxplot(values, **kwds)
```

Comment:
@tacaswell here, these are Series (in a particular case), converting to ndarray makes this work.